config/initializers/statsd.rb

```diff
@@ -226,3 +226,31 @@
 StatsD.increment('iam_ssoe_oauth.inactive_session', 0)
 
 StatsD.increment('iam_ssoe_oauth.auth_type', 0)
+
+# init VEText Push Notifications
+VEText::Service.extend StatsD::Instrument
+VEText::Service.statsd_count_success :register,
+                                     'vetext_push.register'
+VEText::Service.statsd_count_success :get_preferences,
+                                     'vetext_push.get_prefs'
+VEText::Service.statsd_count_success :set_preference,
+                                     'vetext_push.set_pref'
+VEText::Service.statsd_count_success :send_notification,
+                                     'vetext_push.send_notification'
+VEText::Service.statsd_count_success :app_sid,
+                                     'vetext_push.app_lookup'
+
+StatsD.increment('vetext_push.register.success', 0)
```

Comment:
Kinda nitpicky but wondering if you could pull out 'vetext_push' into a STATSD_KEY_PREFIX constant on the VEText::Service class. This seems to be a pattern amongst other metrics in the initializer

Reply:
Will do. Thanks